addOns/llm/src/main/java/org/zaproxy/addon/llm/communication/Confidence.java

```diff
@@ -0,0 +1,47 @@
+package org.zaproxy.addon.llm.communication;
```

Comment:
Needs `./gradlew :aO:llm:sporlessApply` to generate and add the proper license header